src/lib/RPCDefinitions.ts

```diff
@@ -528,3 +528,19 @@ export interface Transaction {
    */
   readonly involvesWatchonly?: boolean;
 }
+
+/**
+ * Result of some estimate fee methods
+ */
+export interface FeeEstimation {
+  /**
+   * The estimated fee the transaction should pay in order to be included within the specified
+   * number of blocks. If the node doesn’t have enough information to make an estimate,
+   * the value -1 will be returned
+   */
+  readonly feerate: number;
```

Comment:
Thoughts on `feeRate` vs `feerate`?

I had to read it several times before I understood what it was supposed to be.

Reply:
@jhodges10 this is mapping the json object, according to [this](https://stackoverflow.com/a/2738375), json should be treated as case-sensitive.

Reply:
Got it! Man it's not easy to read...

Reply:
I know! the alternative would be for us to actually transform the inputs into something else. I like the idea, but it is going to require a lot more work.